server/src/http/express-http-interface.js

```diff
@@ -0,0 +1,39 @@
+function expressHTTPInterfaceFunction({
+  expressServer,
+  cors,
+  jsonSupport,
+  normaliseExpressRequest,
+  handleRequest,
+  ...rest } = {}) {
```

Comment:
I would change `rest` around for something more descriptive like `...params` to avoid having people think it's somehow related to a RESTful API